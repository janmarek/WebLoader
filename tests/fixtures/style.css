@import url(https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,300italic,400italic,600italic);

body {
	background: url(img/boxed-bg.jpg);
}

.greedy-match {
	background: url("img/\"layer1.jpg"), url('img\'/layer2.jpg');
}

.addslashes-if-unquoted {
	background: url(img/'layer1.jpg);
}

.quote-mismatch {
	background: url("img/layer2.jpg');
}

ul {
	list-style-image: url(
		data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4//8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJggg==
	);
}
