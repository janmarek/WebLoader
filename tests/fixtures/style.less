@import 'style2.less';

div.banners {
  .clearFix();
  padding: 0 0 20px;
  margin: 0 10px;
  border-bottom: @borderLight 1px solid;

  & > div {
    float: left;
    width: 610px;
    height: 194px;
    background: @bgBanner;
    margin: 20px 0 0 20px;
    position: relative;

    h3 {
      width: auto;
      color: @textRed;
      font-weight: 600;
      padding: 10px 20px;
      margin: 10px 10px 0 10px;
      text-shadow: 0 2px 0 rgba(0, 0, 0, .3);
      display: inline-block;
      font-size: 24px;
    }

    h3, p {
      color: @white;
    }
    p {
      font-size: 13px;
      max-width: 360px;
      padding: 10px;

      strong {
        font-weight: 600;
      }
    }
  }
}
