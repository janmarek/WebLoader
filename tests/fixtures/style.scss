
@import 'style2.scss';

.navigation {
    ul {
        line-height: 20px;
        color: blue;
        a {
            color: red;
        }
    }
}

.footer {
    .copyright {
        color: silver;
    }
}

