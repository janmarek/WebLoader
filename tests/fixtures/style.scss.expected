.clearFix {
  display: block;
  zoom: 1; }

.navigation ul {
  line-height: 20px;
  color: blue; }
  .navigation ul a {
    color: red; }

.footer .copyright {
  color: silver; }
